tests/test_database.py

```diff
@@ -0,0 +1,74 @@
+import sys
+sys.path.append("/Users/josefinfundin/Dev/Tattoo-Database")
+import unittest
+import database_connector
+import database_manager
```

Comment:
Wouldn't hurt to in general separate into three different sections:

1. Std lib imports
2. 3rd party imports
3. project code

and also alphabetical sorting within each group.

See eg this tool: https://pypi.org/project/reorder-python-imports/

Reply:
Let's discuss this further as I am having an issue with my VSC auto formatting, it put the sys append below all the imports but that won't work. the connector and manager imports are dependent on the append, so let's discuss a solution for this.

Reply:
Suggestions:

1. Add the sys path fix in its own file, and import that one. 
2. Add the sys fix in the \_\_init\_\_.py